pkg/kubectl/cmd/cp.go

```diff
@@ -164,6 +164,9 @@ func (o *CopyOptions) Validate(cmd *cobra.Command, args []string) error {
 }
 
 func (o *CopyOptions) Run(args []string) error {
+	if len(args) < 2 {
+		return fmt.Errorf("Unexpected args: %v, a source and a destination are required", args)
```

Comment:
Just `source and destination are required` will be sufficient.

Reply:
Done,
for approve